backend/src/database/mongo/models/review.ts

```diff
@@ -0,0 +1,33 @@
+import mongoose from 'mongoose';
+
+export interface IReviewLocation {
+  country: string;
+  city?: string;
+}
+export interface IBookReview {
+  bookId: string;
+  location: IReviewLocation;
+  title: string;
+  reviewerName?: string;
```

Comment:
Nit: Name is required in the schema.
```suggestion
  reviewerName: string;
```

Reply:
have delibrately kept name optional to allow for anonymous reviews